Formula/datomic.rb

```diff
@@ -15,6 +15,21 @@ def install
     %w[transactor repl repl-jline rest shell groovysh maven-install].each do |file|
       (bin/"datomic-#{file}").write_env_script libexec/"bin/#{file}", Language::Java.java_home_env
     end
+
+    # create directory for datomic data and logs
+    (var+"lib/datomic").mkpath
+
+    # install free-transactor properties
+    data = var/"lib/datomic"
+    (etc+"datomic").mkpath
+    cp libexec/"config/samples/free-transactor-template.properties", etc/"datomic/free-transactor.properties"
+    inreplace "#{etc}/datomic/free-transactor.properties",
+      "# data-dir=data", "data-dir=#{data}/"
+    inreplace "#{etc}/datomic/free-transactor.properties",
+      "# log-dir=log", "log-dir=#{data}/log"
+
+    # create directory for datomic stdout+stderr output logs
+    (var+"log/datomic").mkpath
```

Comment:
If this is an empty directory, it'll need to be created during the postinstall step to avoid being pruned (i.e., add a `def post_install` section).